core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1552,6 +1554,11 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_set_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_set_rtcp_mux({Endpoint, Call, CallFwd, CCVs}) ->
+    RTCP_MUX = kz_json:is_true([<<"media">>, <<"rtcp_mux">>], Endpoint, ?DEFAULT_RTCP_MUX),
+    {Endpoint, Call, CallFwd, kz_json:set_value(<<"RTCP-MUX">>, RTCP_MUX, CCVs)}.
```

Comment:
@sergey-safarov as commented before, there's no need in sending `rtcp-mux` if its not defined as that will unnecessary increase the payload size that we send over amqp, specially in the case of users with lots of devices or large ring-groups.
lets remove the system default and set a case here as commented earlier.